### DISCUSSION TITLE
Problems with the installation of the plugin

### DISCUSSION THREAD

**Discussion by PhilFalc0ne** [auroraresearchlab/netbox-dns#301] [Q&A] [answered]
Hello community,

I have a problem setting up the plugin.
I have just set up a new Netbox installation according to the howto and wanted to set up the Netbox plugin according to the description.
However, I get the following error message when I run the script "manage.py migrate":

ImportError: cannot import name 'TagFilterField' from 'utilities.forms' (/opt/netbox-3.5.1/netbox/utilities/forms/__init__.py)

My Netbox Version is: 3.5.1
My Python3 Version is: 3.10.6

What am I doing wrong? Can someone give me a tip?

Thanks for your help.

**Comment by peteeckel** [comment, marked as answer]
As it seems, this project has been abandoned.

I created a fork under [peteeckel/netbox-plugin-dns.](https://github.com/peteeckel/netbox-plugin-dns)

The current release 0.18.0 is identical to what would have been release 0.18.0 of this repository. You can install it from PyPI as `netbox-plugin-dns`, and it is compatible with NetBox 3.5.

Sorry for the inconvenience, but the original maintainers have gone away silently, which caused this inconsistency.

**Reply by PhilFalc0ne** [reply]
Hi Peter,
It works like a charm.
Thanks for the quick reply.
